.vscode/tasks.json

```diff
@@ -0,0 +1,21 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "2.0.0",
+    "tasks": [
```

Comment:
Der .vscode folder sollte eigentlich ignoriert werden. Hast du die letzten changes vom master schon mit drin?

Reply:
Eigentlich schon ich habe sie mir rüber ge-pulled in meinen branch...

Reply:
hast du auch `git rebase master` ausgeführt? Der `.gitignore` File müsste am Ende `.vscode/` stehen haben.

Reply:
Wenn der branch nicht up to date ist wird Github den PR nicht mergen lassen.

Reply:
Hmmm... komisch alle anderen Änderungen hab ich drin...

Reply:
Sollte jetzt in Ordnung sein, hab rebase nochmal durchgeführt

Reply:
Immernoch da